roles/mysql/tasks/main.yml

```diff
@@ -30,8 +35,28 @@
     update_cache: true
   when:
     - mysql_server
-  notify:
-    - set root password
+
+- name: MySQL | Restart MySQL
+  ansible.builtin.service:
+    name: mysql
+    state: restarted
+  when: install_app_key.changed
```

Comment:
This one seems like it should be a handler. We might want to restart mysql for other reasons, besides "the key changed".

Reply:
MySQL here fails to work. It needs this to restart so that is isn't listening on `localhost` and listens on all interfaces.